### PR TITLE
Phase2-hcx208 First set of changes for RecHits of HFNose

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

First set of changes to make RecHits for HFNose

#### PR validation:

Tested with standard matrix

#### if this PR is a backport please specify the original PR:

Standard

### CODE REVIEW

**Comment by kpedro88** [comment]
please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@slava77 @perrotta @kpedro88 Could you review this PR at the earliest? I have the second part waiting for this to be integrated. Thanks

File: RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py [hunk 1]
```diff
@@ -60,5 +75,7 @@
 phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCEEConfig , fCPerMIP = fCPerMIP_v10 ) 
 phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCHEFConfig , fCPerMIP = fCPerMIP_v10 )
 
+from Configuration.Eras.Modifier_phase2_hfnose_cff import phase2_hfnose
+phase2_hfnose.toModify( HGCalUncalibRecHit.HGCHFNoseConfig , isSiFE = True )
```
Comment (kpedro88) [inline RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py]: shouldn't this be `True` all the time? Why does the value of `HGCalUncalibRecHit.HGCHFNoseConfig.isSiFE` matter if `phase2_hfnose` is not enabled?

**Comment by kpedro88** [comment]
code-checks

File: RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py (hunk 1 above)
Comment (bsunanda) [reply RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py]: I can make this in my next PR where I make changes to the code which enables RecHits for HFNose

**Comment by kpedro88** [comment]
+upgrade
https://github.com/cms-sw/cmssw/pull/27015#discussion_r290377490 to be addressed in followup PR

**Comment by perrotta** [comment]
+1
- Setup for including RecHits of HFNose
- No changes visible yet in tests outputs

**Comment by fabiocos** [comment]
+1

**Pull request merged**